web/drag.and.drop.upload.php

```diff
@@ -23,8 +23,7 @@
   exit;
 }
 
-# drag and drop window. I did not build this. Still looking for the liscence. Orignial can be found at: https://html5demos.com/dnd-upload/
-
+# drag and drop window. 
```

Comment:
Oh no! Not sure how this file got into this PR... I made the changes you requested on the other PR... So sorry for the confusion